internals/notifier_helpers.py

```diff
@@ -209,3 +209,12 @@ def send_ot_notification(stage: Stage):
   else:
     cloud_tasks_helpers.enqueue_task(
         '/tasks/email-ot-creation-request',params)
+
+
+def send_trial_extended_notification(stage: Stage):
+  """Notify about """
```

Comment:
Incomplete doctoring.

Reply:
Oof, sorry about the careless errors here.